tests/test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import re
+import json
+
+scriptdir = os.path.dirname(os.path.realpath(__file__))
+needledir = os.path.join(scriptdir, "..")
+
+returncode = 0
+
+def error(msg):
+  global returncode
+  returncode = 1
+  print(msg, file=sys.stderr)
```

Comment:
I guess we could just go for pytest code. That should simplify a lot and we have the whole functionality of test runners, nice output, etc.

Reply:
Definitely a thing that could be considered in in the future